src/loader.c

```diff
@@ -198,17 +227,20 @@ static void *imv_loader_bg_new_img(void *data)
       raw_frame_time = *(int*)FreeImage_GetTagValue(tag);
     }
     FreeImage_UnlockPage(mbmp, frame, 0);
-
   } else {
     /* Future TODO: If we load image line-by-line we could stop loading large
      * ones before wasting much more time/memory on them. */
-    FIBITMAP *image = FreeImage_Load(fmt, path, 0);
```

Comment:
This is where we call the `bla_Load` functions, depending on whether we're loading from memory/disk or not.